src/locales/en/translation.json

```diff
@@ -546,6 +546,29 @@
       "emittingTransaction": "Transaction"
     }
   },
+  "consensusEvent": {
+    "governanceProposalExecuted": "Governance Proposal Executed",
+    "governanceProposalFinalized": "Governance Proposal Finalized",
+    "governanceProposalSubmitted": "Governance Proposal Submitted",
+    "governanceVote": "Governance Vote Submitted",
+    "registryEntity": "Entity Registered",
+    "registryNodeUnfrozen": "Node Unfrozen",
+    "registryNode": "Node Registered",
+    "registryRuntime": "Runtime Registered",
+    "registryRuntimeSuspended": "Runtime Suspended",
+    "roothashExecution_discrepancy": "Discrepancy Detected",
+    "roothashExecutorCommitted": "Runtime Executor Committed",
+    "roothashFinalized":  "Runtime Round Finalized",
+    "roothashMessage": "Runtime Message Emitted",
+    "roothashInMsgProcessed": "Incoming Runtime Message Processed",
+    "stakingAllowanceChange": "Allowance Changed",
+    "stakingBurn": "Burn",
+    "stakingEscrowAdd": "Stake Added",
+    "stakingEscrowDebondingStart": "Stake Debonding Started",
+    "stakingEscrowReclaim": "Stake Reclaimed",
```

Comment:
I would copy from sapphire for consistency
```
    "consensusDelegate": "Delegate to consensus",
    "consensusUndelegateStart": "Start to undelegate",
    "consensusUndelegateDone": "Undelegate finished",
```

(tho we may want to switch to stake unstake terminology everywhere)

Reply:
Updated accordingly.